Wikipedia/Code/WMFAppViewController.m

```diff
@@ -182,6 +199,8 @@ - (void)resumeApp {
             [[WMFAlertManager sharedInstance] showAlert:@"You have been notified" sticky:NO dismissPreviousAlerts:NO tapCallBack:NULL];
         }
     }
+
+    [self handleSelectionOfShortcutItemIfNecessary];
```

Comment:
@coreyfloyd iirc we need shouldShowLastReadArticleOnLaunch 'cause we just fall through and let  that logic handle showing the "continue reading" app icon menu item article...